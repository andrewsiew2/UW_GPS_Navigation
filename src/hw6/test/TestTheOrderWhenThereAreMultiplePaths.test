# Tests if the path chosen is always the shortest path when a longer path exists

LoadGraph tester testing.tsv

FindPath tester DARKSEID SPIDERMAN