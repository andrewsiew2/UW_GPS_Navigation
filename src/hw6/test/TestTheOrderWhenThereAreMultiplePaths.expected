# Tests if the path chosen is always the shortest path when a longer path exists

loaded graph tester

path from DARKSEID to SPIDERMAN:
DARKSEID to THANOS via 9000
THANOS to GROOT via 1
GROOT to SPIDERMAN via 2